backend/api/Areas/Property/Mapping/Building/BuildingMap.cs

```diff
@@ -32,6 +32,7 @@ public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.Building, Model.BuildingModel>()
                 .Map(dest => dest.Id, src => src.Id)
+                .Map(dest => dest.PropertyTypeId, src => src.PropertyTypeId)
```

Comment:
building icon clicks on the map were showing up as parcels, because the mapping was skipping this field and always returning 0.